ansible/roles/edu-sharing/tasks/classification-keyword.yml

```diff
@@ -45,21 +45,21 @@
 
 - name: unzip classification keywords gz format
   command:
-    chdir: '/var/www/classification-keywords/'
-    cmd: 'gunzip classification_keywords.xml.gz'
-    creates: '/var/www/classification-keywords/classification_keywords.xml'
+    chdir: '/var/www/{{edu_classification_keyword_path | dirname  }}'
+    cmd: 'gunzip {{edu_classification_keyword_path | basename }}.gz'
+    creates: '/var/www/{{edu_classification_keyword_path}}'
   when: not classification_keywords_xml_file_exist.stat.exists
   tags:
   - classification-keyword
 
 - name: remove gz file
-  file: path="/var/www/classification-keywords/classification_keywords.xml.gz" state=absent
+  file: path="/var/www/{{edu_classification_keyword_path | basename }}.gz" state=absent
```

Comment:
shouldn't this be the complete path instead of the basename?

Reply:
I think yes